src/app/organizations/manage/user-add-edit.component.html

```diff
@@ -255,8 +264,8 @@ <h3 class="mt-4 d-flex">
                     <i class="fa fa-spinner fa-spin" title="{{'loading' | i18n}}" aria-hidden="true"></i>
                     <span>{{'save' | i18n}}</span>
                 </button>
-                <button type="button" class="btn btn-outline-secondary"
-                    data-dismiss="modal">{{'cancel' | i18n}}</button>
+                <button type="button" class="btn btn-outline-secondary" data-dismiss="modal">{{'cancel' |
```

Comment:
This formatting change is breaking the line in a bad spot IMHO... it'll "work" but it looks really wonky terminating a line with a pipe in the middle of an angular content directive.

Reply:
I concur - will update!

Reply:
Latest updates should have this fixed for you @cscharf 

👍